config/config.go

```diff
@@ -47,6 +49,14 @@ type RedisConf struct {
 	MasterName string
 }
 
+type SpannerConfig struct {
+	Project         string `mapstructure:"project" validate:"required"`
+	Instance        string `mapstructure:"instance" validate:"required"`
+	Database        string `mapstructure:"db" validate:"required"`
+	CredentialsFile string `mapstructure:"credentials_file"`
+	TableName       string
```

Comment:
```suggestion
	TableName       string `mapstructure:"table_name"`
```

Reply:
adopted

Reply:
Hi, @Napoleon0621 

You can click the `Resolve conversation` button after resolving and reviewers can have a look again after all conservations are resolved.